packages/verification-pool-api/package.json

```diff
@@ -28,7 +28,7 @@
     "ethereumjs-util": "^5.2.0",
     "express": "^4.16.4",
     "firebase-admin": "^7.0.0",
-    "firebase-functions": "^2.2.0",
+    "firebase-functions": "^2.3.1",
```

Comment:
Please redeploy the pool to ensure it still works after this update. Deploying to Int should be fine as we're not using it atm

Reply:
test deployed it. It works fine.